CHANGELOG.adoc

```diff
@@ -4,13 +4,16 @@
 
 === Shapes
 
+- Add Impact Analysis Before Tool Execution in Trees
```

Comment:
no need to use an upper case letter for each word

Reply:
fixed